mcodingbot/plugins/highlights.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import annotations
+import asyncio
+from collections import defaultdict
+import contextlib
+
+import crescent
+import hikari
+from mcodingbot.utils import Plugin, Context
+from mcodingbot.database.models.user import Word, User
+
+plugin = Plugin()
+highlights_group = crescent.Group("highlights")
+highlights_cache: dict[str, list[hikari.Snowflake]] = defaultdict(list)
+
+
+def _cache_highlight(word: str, *user_ids: hikari.Snowflake) -> None:
+    for user_id in user_ids:
+        highlights_cache[word].append(user_id)
```

Comment:
.extend()

Reply:
`.append` is definitely the right one here

Reply:
highlights_cache[word].extend(user_ids), instead of a for-loop